heimdall/scripts/view_container.py

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/python
+#!/usr/bin/env python
 
 import argparse
-from heimdall import view_container
+from .. import view_container
```

Comment:
I am not quite familiar with exporting scripts with `setup.py`, but If I understand the mechanism correctly, it copies the script to the `bin` of the environment and replaces the shebang.
So how will this work with relative imports?

Reply:
It puts a little script into the virtualenv's bin directory which basically arranges for the specified function to be called in the context of the environment and library; I'm pretty sure relative imports work fine for that reason.